model/file.py

```diff
@@ -36,3 +47,35 @@ def get(cls, identifier):
             File.md5 == identifier,
             File.ssdeep == identifier,
             File.humanhash == identifier))
+
+    @classmethod
+    def get_or_create(cls, file, parent=None, metakeys=None, share_with=None):
+        file.stream.seek(0, os.SEEK_END)
```

Comment:
trochę yolo to latanie po streamie tam i z powrotem :thinking:. Ale no i guess (tak czy inaczej musi być wczytany do RAMu w całości w tym przypadku)

Reply:
Sometimes there is temporary file under this stream... but yeah, it's completely loaded anyway.